README.md

```diff
@@ -49,37 +65,34 @@ npm run build
 
 When working with non-source-tracked orgs, use the commands `SFDX: Deploy Source to Org` (VS Code) or `sfdx force:source:deploy` (Salesforce CLI) and `SFDX: Retrieve Source from Org` (VS Code) or `sfdx force:source:retrieve` (Salesforce CLI). The `Push` and `Pull` commands work only on orgs with source tracking (scratch orgs).
 
-### 2b. Pushing the code in a Scratch Org
+### 3b. Pushing the code in a Scratch Org
 
 When working with source-tracked orgs, use the commands `SFDX: Push Source to Org` (VS Code) or `sfdx force:source:push` (Salesforce CLI) and `SFDX: Pull Source from Org` (VS Code) or `sfdx force:source:pull` (Salesforce CLI). Do not use the `Retrieve` and `Deploy` commands with scratch orgs.
 
-### 2c. Installing the app using an Unlocked Package
+### 3c. Installing the app using an Unlocked Package
 
 TBD
 
-### 3. How to enable the Case Flow in your community
-
-1. In the class CaseAssistEndpoint. Change this line to add your case assist ID:
-   `public static final String CASE_ASSIST_ID = 'YOUR_CASE_ASSIST_ID';`
-1. In your Salesforce community, drag the Lightning Flow component onto a Community page and select the Case_Assist_Demo_Flow shipped with this repository.
-1. In the published version of your community, fill in the subject and description fields **(minimum of 10 characters in the description)** in the first screen to see the Related Category section appear and get predictions for the Case Reason field.
-
-### 4. Sending analytics with the Case Flow
+### 4. How to enable the Case Flow in your community
 
-Make sure you drag and drop the component AnalyticsBeacon somewhere in your community in order for the Case Flow to send analytics events.
-We recommend dropping it in your community header or footer region (It is invisible to the end user).
+1. In your Salesforce community, drag the Lightning Flow component onto a Community page and select the Case_Assist_Recommended_Flow or the Case_Assist_Demo_Flow shipped with this repository.
+2. After selecting the name of the flow two inputs labeled caseAssistId and engineId appear. Fill the caseAssistId input with your [Case Assist Id](https://docs.coveo.com/en/3328/service/manage-case-assist-configurations#retrieving-a-case-assist-id "Case Assist Id") retrieved from your Case assist Configuration and fill the enginId input with an appropriate name for the engine that the Quantic components will register to.
+3. In the published version of your community, fill in the subject and description fields in the first screen.
+4. Proceed to the next screens to see the proposed predictions for classifying your case and get documents suggestions that will potentially resolve your case before it is created.
 
-If you do not have it you will see that error in the console : `Analytics Beacon cannot be found, include the component in your page to send events.`.
-
-## Dev, Build and Test
+### Dev, Build and Test
 
 1. Run `npm i`
 1. Run `npm run build`
```

Comment:
I would also capitalize the title below:
`Developing locally` -> `Developing Locally`